app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @recipe.name %></h1>
+<p>Recipe: <%= @recipe.name %></p>
+<p>Recipe: <%= @recipe.complexity %></p>
+<p>Recipe: <%= @recipe.genre %></p>
+
+<h1><%= "List of Ingredients for recipe" %></h1>
```

Comment:
This is close but we want to iterate vs calling `.first`. So something like:
```ruby
<% @recipe.ingredients.each do |ingredient| %>
   <li><%= ingredient.name %></li>
<% end %>
```
For testing purposes we could also wrap this with an id so that we can use within blocks for our tests.